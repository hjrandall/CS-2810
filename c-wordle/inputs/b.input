a(r)i(s)(e)
[r]o(u)t(e)
