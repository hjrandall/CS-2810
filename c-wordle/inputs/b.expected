173: rubes
171: ruses
168: rudes
168: rules
168: rumes
168: runes
147: ruers
137: rebus
135: resus
